components/brave_ads/browser/ads_service_impl_unittest.cc

```diff
@@ -6,19 +6,18 @@
 #include <map>
 
 #include "base/files/scoped_temp_dir.h"
-#include "extensions/browser/test_event_router.h"
```

Comment:
This is the only non-cosmetic change here. This can't be included on Android but since it seemed not to be used on any platform, I just removed it.